_includes/contact_phone.html

```diff
@@ -0,0 +1,6 @@
+<h3 id="call-us">{{ site.data.[page.lang].settings.contact_page.call_us.title }}</h3>
```

Comment:
`nit` The other titles on the bottom half of the page are using `h2` tags.  My OCD is saying we should use the same size or tighten up the spacing a little by using `br` instead of a `p` section.  Won't be offended if this merges without changing, it just looks a little odd to me 


![Screen Shot 2022-08-23 at 4 08 43 PM](https://user-images.githubusercontent.com/10132008/186256009-faeea67d-6c27-4c66-aa45-b5b8fdcc8b32.png)

Reply:
The idea is that "Call us" is a part of the "Get in touch" h2. It's easier to see when it's laid out in the design file, but I agree, maybe a little awkward in situ. 

https://www.figma.com/file/lipEotvTZ7I6xskooHKom2/Support-Phone-Number?node-id=0%3A1

Reply:
Design and product agreed that it's a bit awkward, but said it's good enough to stick to the predefined styles (and minimize the size) for the pilot. They'll readdress if it causes an issue in the future.